ARCollectionViewMasonryLayout.m

```diff
@@ -122,6 +126,9 @@ - (void)setupLayoutWithStaticDimension:(CGFloat)staticDimension andVariableDimen
 {
     NSAssert(self.rank > 0, @"Rank for ARCollectionViewMasonryLayout should be greater than 0.");
     NSAssert(self.collectionView.numberOfSections == 1, @"ARCollectionViewmMasonry doesn't support multiple sections.");
+
+    UICollectionViewFlowLayoutInvalidationContext *invalidationContext = [[UICollectionViewFlowLayoutInvalidationContext alloc] init];
```

Comment:
previously we sent the header footer with every scroll, now we keep track of when we need to send them via invalidation contexts.